apps/src/turtle/turtle.js

```diff
@@ -394,6 +396,10 @@ class Visualization {
     }
   }
 
+  pointTo(degrees) {
+    this.setHeading(degrees + ABSOLUTE_DIRECTION);
```

Comment:
`DEFAULT_DIRECTION` would make more sense here, and that constant already exists (fortunately with the same value).

Reply:
Fixed.  Thanks